hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -101,19 +144,56 @@ def lambda_handler(event, context):
 
 
 # TODO move to utils library
-def check_db_for_matches():
+def gen_matches() -> t.List:
```

Comment:
(ignorable): The strictest typing modes will keep asking you to make this more specific List[Dict[str, Any]]